backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po

```diff
@@ -94,11 +94,11 @@ msgstr ""
 
 #: tracim_backend/lib/core/workspace.py:400
 msgid "Workspace"
-msgstr "Shared space"
+msgstr "Space"
```

Comment:
Should I send push a commit commit to fix the string in the python file?

 Otherwise, here are the changes to make in `tracim_backend/lib/core/workspace.py`:

```python
        query = self._base_query_without_roles().filter(
            Workspace.label.ilike("{0}%".format(_("Workspace")))
        )

        return _("Workspace {}").format(query.count() + 1)
```

Becomes:

```python
        query = self._base_query_without_roles().filter(
            Workspace.label.ilike("{0}%".format(_("Space")))
        )

        return _("Space {}").format(query.count() + 1)
```

Reply:
I don't see any drawback to do so as soon as you correctly updated the translation scripts (for all lang because the key as changed !), but as we do currently use the term "workspace" everywhere in backend and we do not decide to change it, i don't see any advantage to do a very partial conversion now.

Do as you want, this seems to me not a really important point.

Reply:
It seems like a string that is exposed to the user, so we should definitely update this I think.

Reply:
You're right, this can cause issue for uncompleted new translation (english case is handled here).